map_xspacing: 0.5 # spacing between rows
map_yspacing: 0.5 # spacing between column

tagmap:
  {
    points:
    [
      {tag_id: 0, row: 0, col: 0},
      {tag_id: 1, row: 0, col: 1},
      {tag_id: 2, row: 0, col: 2},
      {tag_id: 3, row: 0, col: 3},
      {tag_id: 4, row: 0, col: 4},
      {tag_id: 5, row: 0, col: 5},
      {tag_id: 6, row: 0, col: 6},
      {tag_id: 7, row: 0, col: 7},
      {tag_id: 8, row: 0, col: 8},
      {tag_id: 9, row: 0, col: 9},

      {tag_id: 10, row: 1, col: 0},
      {tag_id: 11, row: 1, col: 1},
      {tag_id: 12, row: 1, col: 2},
      {tag_id: 13, row: 1, col: 3},
      {tag_id: 14, row: 1, col: 4},
      {tag_id: 15, row: 1, col: 5},
      {tag_id: 16, row: 1, col: 6},
      {tag_id: 17, row: 1, col: 7},
      {tag_id: 18, row: 1, col: 8},
      {tag_id: 19, row: 1, col: 9},

      {tag_id: 20, row: 2, col: 0},
      {tag_id: 21, row: 2, col: 1},
      {tag_id: 22, row: 2, col: 2},
      {tag_id: 23, row: 2, col: 3},
      {tag_id: 24, row: 2, col: 4},
      {tag_id: 25, row: 2, col: 5},
      {tag_id: 26, row: 2, col: 6},
      {tag_id: 27, row: 2, col: 7},
      {tag_id: 28, row: 2, col: 8},
      {tag_id: 29, row: 2, col: 9},

      {tag_id: 30, row: 3, col: 0},
      {tag_id: 31, row: 3, col: 1},
      {tag_id: 32, row: 3, col: 2},
      {tag_id: 33, row: 3, col: 3},
      {tag_id: 34, row: 3, col: 4},
      {tag_id: 35, row: 3, col: 5},
      {tag_id: 36, row: 3, col: 6},
      {tag_id: 37, row: 3, col: 7},
      {tag_id: 38, row: 3, col: 8},
      {tag_id: 39, row: 3, col: 9},

      {tag_id: 40, row: 4, col: 0},
      {tag_id: 41, row: 4, col: 1},
      {tag_id: 42, row: 4, col: 2},
      {tag_id: 43, row: 4, col: 3},
      {tag_id: 44, row: 4, col: 4},
      {tag_id: 45, row: 4, col: 5},
      {tag_id: 46, row: 4, col: 6},
      {tag_id: 47, row: 4, col: 7},
      {tag_id: 48, row: 4, col: 8},
      {tag_id: 49, row: 4, col: 9},

      {tag_id: 50, row: 5, col: 0},
      {tag_id: 51, row: 5, col: 1},
      {tag_id: 52, row: 5, col: 2},
      {tag_id: 53, row: 5, col: 3},
      {tag_id: 54, row: 5, col: 4},
      {tag_id: 55, row: 5, col: 5},
      {tag_id: 56, row: 5, col: 6},
      {tag_id: 57, row: 5, col: 7},
      {tag_id: 58, row: 5, col: 8},
      {tag_id: 59, row: 5, col: 9},

      {tag_id: 60, row: 6, col: 0},
      {tag_id: 61, row: 6, col: 1},
      {tag_id: 62, row: 6, col: 2},
      {tag_id: 63, row: 6, col: 3},
      {tag_id: 64, row: 6, col: 4},
      {tag_id: 65, row: 6, col: 5},
      {tag_id: 66, row: 6, col: 6},
      {tag_id: 67, row: 6, col: 7},
      {tag_id: 68, row: 6, col: 8},
      {tag_id: 69, row: 6, col: 9},

      {tag_id: 70, row: 7, col: 0},
      {tag_id: 71, row: 7, col: 1},
      {tag_id: 72, row: 7, col: 2},
      {tag_id: 73, row: 7, col: 3},
      {tag_id: 74, row: 7, col: 4},
      {tag_id: 75, row: 7, col: 5},
      {tag_id: 76, row: 7, col: 6},
      {tag_id: 77, row: 7, col: 7},
      {tag_id: 78, row: 7, col: 8},
      {tag_id: 79, row: 7, col: 9},

      {tag_id: 80, row: 8, col: 0},
      {tag_id: 81, row: 8, col: 1},
      {tag_id: 82, row: 8, col: 2},
      {tag_id: 83, row: 8, col: 3},
      {tag_id: 84, row: 8, col: 4},
      {tag_id: 85, row: 8, col: 5},
      {tag_id: 86, row: 8, col: 6},
      {tag_id: 87, row: 8, col: 7},
      {tag_id: 88, row: 8, col: 8},
      {tag_id: 89, row: 8, col: 9},

      {tag_id: 90, row: 9, col: 0},
      {tag_id: 91, row: 9, col: 1},
      {tag_id: 92, row: 9, col: 2},
      {tag_id: 93, row: 9, col: 3},
      {tag_id: 94, row: 9, col: 4},
      {tag_id: 95, row: 9, col: 5},
      {tag_id: 96, row: 9, col: 6},
      {tag_id: 97, row: 9, col: 7},
      {tag_id: 98, row: 9, col: 8},
      {tag_id: 99, row: 9, col: 9},
    ],
  }
